From 77c9f87dc2b3ad0854a678e234e22dfb31902b82 Mon Sep 17 00:00:00 2001
From: Alexander Kanavin <alex.kanavin@gmail.com>
Date: Wed, 31 Jan 2018 15:50:38 +0200
Subject: [PATCH] help/meson.build: disable the use of yelp

In particular this avoids calling itstool which oe-core doesn't provide.

Upstream-Status: Inappropriate [oe-core specific]
Signed-off-by: Alexander Kanavin <alex.kanavin@gmail.com>

---
 help/meson.build | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/help/meson.build b/help/meson.build
index ff324dd..f5e421d 100644
--- a/help/meson.build
+++ b/help/meson.build
@@ -31,7 +31,7 @@ help_media = [
   'media/org.gnome.Epiphany.svg'
 ]
 
-gnome.yelp(meson.project_name(),
-  sources: help_files,
-  media: help_media,
-)
+#gnome.yelp(meson.project_name(),
+#  sources: help_files,
+#  media: help_media,
+#)
